### PR TITLE
Add Prettier to the contract test sequence

### PR DESCRIPTION
**Pull request by albertofaria**

### CODE REVIEW

File: truffle/test/marketplace_test.js [hunk 1]
```diff
@@ -1,255 +1,378 @@
 /* -------------------------------------------------------------------------- */
 
 const TuiChainController = artifacts.require("TuiChainController");
-const TuiChainMarket     = artifacts.require("TuiChainMarket");
-const TuiChainLoan       = artifacts.require("TuiChainLoan");
-const TuiChainToken      = artifacts.require("TuiChainToken");
```
Comment (nelsontss) [inline truffle/test/marketplace_test.js:6]: Align the assignments. It's good pratice

File: truffle/contracts/TuiChainController.sol [hunk 2]
```diff
@@ -76,9 +74,10 @@ contract TuiChainController is Ownable
      *     atto-Dai per nano-Dai
      */
     function setMarketFee(uint256 _marketFeeAttoDaiPerNanoDai)
-        external onlyOwner
+        external
+        onlyOwner
     {
-        market.setFee({ _feeAttoDaiPerNanoDai: _marketFeeAttoDaiPerNanoDai });
+        market.setFee({_feeAttoDaiPerNanoDai: _marketFeeAttoDaiPerNanoDai});
```
Comment (nelsontss) [inline truffle/contracts/TuiChainController.sol:80]: The space after a curly brackets it's good pratice too.

File: truffle/test/marketplace_test.js (hunk 1 above)
Comment (albertofaria) [reply truffle/test/marketplace_test.js:6]: AFAICT, Prettier unfortunately has no option for this. Although I also prefer to align assignments, I'd argue that it's more a matter of taste than of good practice.

File: truffle/contracts/TuiChainController.sol (hunk 2 above)
Comment (albertofaria) [reply truffle/contracts/TuiChainController.sol:80]: I also couldn't find an option for this one (formatting for Solidity files uses https://github.com/prettier-solidity/prettier-plugin-solidity).

**Review by nelsontss — APPROVED** [review]

**Review by joaop21 — APPROVED** [review]

**Pull request merged**